bindings/go/plugin/manager/types/types.go

```diff
@@ -0,0 +1,103 @@
+package types
```

Comment:
I am missing versioning of these types. What is your plan here? Also I was expecting the types to be colocated with the individual contracts. Currently they would be all in this file which could get messy

Reply:
Absolutely. And I was planning on creating folders for the versions, like the the bindings do. so `v1`... etc.

Reply:
Then this should be replaced accordingly I think, as these types need to be checked in accordingly right?